docs/content/reference/resource-schema/core-schema/container-schema/snippets/base-container.yaml

```diff
@@ -16,4 +16,4 @@ spec:
     spec:
       containers:
         - name: frontend
-          image: radiusdev.azurecr.io/magpie:latest
\ No newline at end of file
+          image: radiusdev.ghcr.io/magpie:latest
```

Comment:
Need help identifying if this package called `magpie` maps to `magpieo`, `ghcr.io/radius-project/magpiego:latest`.